generator.js

```diff
@@ -15,8 +23,8 @@ module.exports = api => {
 
   api.extendPackage({
     dependencies: {
-      'single-spa-vue': '^1.5.2',
-      'systemjs-webpack-interop': '^1.1.0',
+      'single-spa-vue': '^1.9.0',
+      'systemjs-webpack-interop': '^2.1.2',
```

Comment:
This is a safe upgrade. See https://github.com/joeldenning/systemjs-webpack-interop/releases/tag/v2.0.0